docs/design-docs/multi-protocol-cidr.md

```diff
@@ -66,6 +66,14 @@ The necessary changes would be localized within the GCP-specific implementation
   * gce.EnsureLoadBalancerDeleted: This function would need to be updated to iterate through all potential forwarding rules based on the new naming scheme and delete them.  
   * **Resource Naming Functions:** Helper functions that generate names for GCP resources would need to be adapted to produce the protocol-specific forwarding rule names.
 
+### 5. Production Readiness
+
+For ease of introduction, we will implement a feature flag for the support.  If the feature flag is not set,
```

Comment:
is not really an error , is it? the status gets updated to indicate one protocol is not supported